tracing/src/main/java/com/palantir/tracing/Traces.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.tracing;
+
+import com.google.common.base.Strings;
+
+public final class Traces {
+
+    public static final String TRACE_HEADER = "trace-id";
```

Comment:
Ah, ok. Happy to switch headers. It looks like those take 64bit identifiers instead of arbitrary strings, which makes one of the things here less ideal -- we wanted the ability to name a trace explicitly (rather than use a generated identifier), so perhaps we should pass that separately?